### DISCUSSION TITLE
Modal with custom width

### DISCUSSION THREAD

**Discussion by LauGroupie95** [buildo/bento-design-system#541] [Q&A] [answered]
Hi @veej and @gabro,
can we have the chance to set the width of a modal?

Your colleague designed a modal with width 1016 px, but the biggest dimension you provided in the configuration of the modal is "wide" (whose size is 1000 px). 

![image](https://user-images.githubusercontent.com/63547770/223729855-f540a706-ac6c-413c-ae56-b45bf105412c.png)

We used the x-overflow for now, but it could be useful to personalize the max width of the modal. 

![image](https://user-images.githubusercontent.com/63547770/223740140-2df09faa-52bb-4642-8eff-c3e603911eb9.png)

Thank you in advance

**Comment by veej** [comment, marked as answer]
Hi @LauGroupie95,

it might make sense in general to allow custom widths for a modal. I'll discuss this internally to collect opinions on the matter.

In the meanwhile, for you specific case, you may consider to configure a different width for Wide modals on the whole project. 
Bento, indeed, allows to globally configure the width associated to each modal size (small|medium|large|wide), as per documentation: https://zeroheight.com/1424b583b/p/238cdb-modal/b/58f106. I already checked with Agnese, and it seems ok to do this globally, as the difference is very small and it should not be an issue to apply it also to other wide modals in the project.

**Reply by LauGroupie95** [reply]
Thank you @veej , I modified the configurations for the wide modal in the index of the design system and now it's perfect!
Please, keep me updated when you'll insert the custom widths for modals. Bye :)
